src/imagery/i.sam2/i.sam2.py

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/env python3
+
+############################################################################
+#
+# MODULE:       i.sam2
+# AUTHOR:       Corey T. White, OpenPlains Inc.
+# PURPOSE:      Uses the SAMGeo model for segmentation in GRASS GIS.
+# COPYRIGHT:    (C) 2023-2024 Corey White
+#               This program is free software under the GNU General
+#               Public License (>=v2). Read the file COPYING that
+#               comes with GRASS for details.
+#
+#############################################################################
+
+# %module
+# % description: Integrates SAMGeo model for segmentation in GRASS GIS.
+# % keyword: raster
+# % keyword: segmentation
+# % keyword: deep learning
```

Comment:
```suggestion
# % keyword: imagery
# % keyword: segmentation
# % keyword: object recognition
# % keyword: deep learning
```

i.* -> imagery for manual creation consistency

(now similar to keywords in https://grass.osgeo.org/grass84/manuals/i.segment.html)